### PR TITLE
Test information displayed in columns on Run Overview page

### PR DESCRIPTION
**Pull request by jadecarino**
Test info is now displayed in columns on the Run Overview page using RunOverview, RunDetail and DateTime components. Styling to do.

### CODE REVIEW

File: src/app/date-time/date-time.component.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+@Component({
+  selector: 'app-date-time',
+  templateUrl: './date-time.component.html',
+  styleUrls: ['./date-time.component.css']
+})
+export class DateTimeComponent implements OnInit {
+
+  @Input() title: Object;
+
+  @Input() value: Object;
```
Comment (CJMeek) [inline src/app/date-time/date-time.component.ts]: title and value should be strings not objects

File: src/app/run-page/components/run-detail/run-detail.component.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+@Component({
+  selector: 'app-run-detail',
+  templateUrl: './run-detail.component.html',
+  styleUrls: ['./run-detail.component.css']
+})
+export class RunDetailComponent implements OnInit {
+
+  @Input() title: Object;
+
+  @Input() value: Object;
```
Comment (CJMeek) [inline src/app/run-page/components/run-detail/run-detail.component.ts]: Same here

File: src/app/run-page/components/run-overview/run-overview.component.html [hunk 3]
```diff
@@ -1,3 +1,14 @@
...
+<app-run-detail [title] = "'Short Name:'" [value] = "testShortName"></app-run-detail>
+<app-run-detail [title] = "'Bundle:'" [value] = "bundle"></app-run-detail>
+<app-run-detail [title] = "'Test Name:'" [value] = "testName"></app-run-detail>
+<app-run-detail [title] = "'Requestor:'" [value] = "requestor"></app-run-detail>
+<app-date-time [title] = "'Run Requested:'" [value] = "queued"></app-date-time>
+<app-date-time [title] = "'Run Started:'" [value] = "startTime"></app-date-time>
+<app-date-time [title] = "'Run Finished:'" [value] = "endTime"></app-date-time>
 <div id = "container">
     <app-methods [testMethods] = "testMethods"></app-methods>
 </div>
```
Comment (CJMeek) [inline src/app/run-page/components/run-overview/run-overview.component.html]: Wrap all of this content in the html in this container

**Review by CJMeek — CHANGES_REQUESTED** [review]

**Comment by jadecarino** [comment]
rerun tests

**Review by Michael-Baylis — DISMISSED** [review]

**Review by Michael-Baylis — APPROVED** [review]

**Review by CJMeek — APPROVED** [review]

**Pull request merged**